pkg/agent/datapath/multiBridgeDatapath.go

```diff
@@ -849,11 +849,6 @@ func (datapathManager *DpManager) AddEveroutePolicyRule(rule *EveroutePolicyRule
 			return nil
 		}
 		log.Infof("Rule already exists. update old rule: {%+v} to new rule: {%+v} ", ruleEntry.EveroutePolicyRule, rule)
-
```

Comment:
It seems that check the rule exists or not is unnecessary. Can you delete these code?

Reply:
Each EveroutePolicyRule may relate to several securityPolicies. This party we cache the corresponding relationships.